packages/fast-components-react-base/src/media/Media.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import {IMediaProps, MediaHTMLTags} from "./Media.props";
+import {IMediaClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/* tslint:disable-next-line */
+class Media extends Foundation<IMediaProps & IManagedClasses<IMediaClassNameContract>,  React.HTMLAttributes<HTMLImageElement | HTMLPictureElement>, {}> {
+    protected handledProps: HandledProps<IMediaProps & IManagedClasses<IMediaClassNameContract>> = {
+        managedClasses: void 0,
+        alt: void 0
+    };
+
+    generatePicture(): React.ReactElement<HTMLPictureElement> {
+         let spaceSplit = /\s* /;
```

Comment:
Let's talk as a team about this implementation. I think that strings could be a difficult pattern to use if teams want to create a higher order component, or create repeatable patterns with their code. I think that expecting an array of objects that translate to viewports would likely be an easier API for teams to consume and work with.